### PR TITLE
Rename css variables

### PR DESCRIPTION
**Pull request by blackinno**
Card components
- set color for a border-bottom from props
Rename css variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,29 @@ $ npm run storybook
...
 $ npm run build-storybook
 ```
+
+## Style root color components
+
+```css
+:root {
+  --primary: #491e8c;
+  --secondary: #3b1873;
+  --dark-purple: #210e41;
```
Comment (antho1404) [inline README.md]: Maybe we could rename the purple in all these variable
```
--primary-dark
--primary-very-dark
--primary-light
--primary-very-light
```

File: README.md [hunk 2]
```diff
@@ -12,3 +12,29 @@ $ npm run storybook
...
+  --light-grey: #f2f0f7;
+  --dark-grey: #3c3940;
+  --Black: #000000;
+  --White: #ffffff;
+
+  --Error: #f44336;
+  --Success: #4bb543;
+  --Info: #1e88e5;
+
+  --Sale: #2e1359;
```
Comment (antho1404) [inline README.md]: Do we really need these 4 variables?
This is related to the ATD and not the components. I'm not sure we are even using them in any components

Comment (blackinno) [reply README.md]: Sorry, I forget to remove them.

File: README.md (hunk 1 above)
Comment (blackinno) [reply README.md]: Okay, I will change it.

Comment (blackinno) [reply README.md]: Fixed.

File: storybook/style.css [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
+  --primary-very-dark: #0e061c;
+  --primary-dark: #210e41;
+  --primary: #491e8c;
+  --secondary: #3B1873;
+  --primary-light: #7e61ae;
+  --primary-very-light: #bdafd6;
+  --dark-grey: #3c3940;
+  --light-grey: #f2f0f7;
   --White: #ffffff;
   --Wallets-Sale: #2e1359;
```
Comment (antho1404) [inline storybook/style.css]: still some here, are these useful?

**Comment by antho1404** [comment]
I think the renaming to index should go on another PR, if you can revert your last commit and put it on another PR like that we can merge this one

**Comment by blackinno** [comment]
I have reverted to the last commit  and fixed unused css variables

**Pull request merged**